extension/typedefs/shopify/GetRootCategoriesResponse.d.ts

```diff
@@ -0,0 +1,16 @@
+interface CategoryChild {
+    id: string
+    name: string
+    productCount: number
+    imageUrl: string
+    childrenCount: number
+}
+
+interface GetRootCategoriesResponse {
```

Comment:
This should probably something like this:

interface RootCategories {
    id: string
    name: string
    productCount: number
    imageUrl: string|null
    childrenCount: number
    children?: CategoryChild[]
}

interface GetRootCategoriesResponse {
    categories: RootCategories[]
}

Reply:
done